### PR TITLE
docs for setting price BB and Platforms

### PR DESCRIPTION
**Pull request by PolinaSadykova**
Make the page more structured, added information about setting price

### CODE REVIEW

File: docs/meshstack.billing-configuration.md [hunk 1]
```diff
@@ -437,128 +510,42 @@ let example
...
+- **Private Cloud billing**: Partners can define products and usage rates in any currency.
+- **Public Cloud billing**: meshStack preserves the original currency when importing cost and consumption data from the cloud provider.
+- *OSB Services** (old meshMarketplace): Service owners can define prices for their services using the OSB Catalog in any currency.
+- **Tenant Usage Reports**: Tenant usage reports support line items in multiple currencies and aggregates net amounts in each currency separately.
+- **Chargeback Statements**: Chargeback statements support multiple currencies and aggregate net amounts in each currency separately.
+- **Tenant Fees**: Partners can define tenant fees in any currency.
+- **Discounts**: Discounts support any currency. Discounts calculated from a source amount (e.g. percentage discounts) use the same currency as the source amount.
 
-### Configuration example
+The following product functionalitities do currently not support multi-currency scenarios:
```
Comment (nroi) [inline docs/meshstack.billing-configuration.md:542]:
```suggestion
The following product functionalities do currently not support multi-currency scenarios:
```

File: docs/meshstack.billing-configuration.md [hunk 2]
```diff
@@ -437,128 +510,42 @@ let example
...
-## Chargeback
-
-> Chargeback is the process of allocating IT cost to consumers and feeding it into the company-wide finance and controlling processes.
+## Currency Conversion
 
-Each project in meshStack is associated with a Chargeback Account. meshStack periodically generates [chargeback statements](meshcloud.project-metering.md#chargeback-statements).
+In order to simplify multi-cloud governance, the best practice is to have single chargeback currency. meshStack simplifies the chargeback process with automatic currency conversion capability, namely regardless of the original currency from cloud providers like AWS chargeback statements will be converted to a single currency - Euro. 
 
-The attributes that shall be part of the billing info on the chargeback statements can be configured.
+You have two options based on your [plan](https://www.meshcloud.io/en/pricing/) to enable currency conversion for chargebacks and usage reports:
```
Comment (nroi) [inline docs/meshstack.billing-configuration.md:518]: d: This link (https://www.meshcloud.io/en/pricing/) just contains a form for a price inquiry, it provides no information about the plan. Not sure what value this link should provide in this context.

File: docs/meshstack.billing-configuration.md [hunk 3]
```diff
@@ -437,128 +510,42 @@ let example
...
-You can define products based on this resource type to charge fees based on the Landing Zone that is in use by a meshTenant
-or to simply charge a fee on the meshTenant itself.
 
-## Chargeback
-
-> Chargeback is the process of allocating IT cost to consumers and feeding it into the company-wide finance and controlling processes.
+## Currency Conversion
 
-Each project in meshStack is associated with a Chargeback Account. meshStack periodically generates [chargeback statements](meshcloud.project-metering.md#chargeback-statements).
+In order to simplify multi-cloud governance, the best practice is to have single chargeback currency. meshStack simplifies the chargeback process with automatic currency conversion capability, namely regardless of the original currency from cloud providers like AWS chargeback statements will be converted to a single currency - Euro.
```
Comment (nroi) [inline docs/meshstack.billing-configuration.md:516]: d: "**a** single chargeback currency"

```suggestion
In order to simplify multi-cloud governance, the best practice is to have a single chargeback currency. meshStack simplifies the chargeback process with automatic currency conversion capability, namely regardless of the original currency from cloud providers like AWS chargeback statements will be converted to a single currency - Euro. 
```

**Review by nroi — APPROVED** [review]
just a few remarks.

File: docs/meshstack.billing-configuration.md (hunk 2 above)
Comment (PolinaSadykova) [reply docs/meshstack.billing-configuration.md:518]: you are right:)

**Pull request merged**